src/_data/books.yml

```
@@ -76,3 +76,10 @@
   cover: programming-flutter.jpg
   link: "https://pragprog.com/book/czflutr"
   desc: "Native, Cross-Platform Apps the Easy Way. Doesn't require previous Dart knowledge."
+- title: 'Flutter Libraries we love'
+  authors:
+    - Codemagic
+  publisher: Codemagic
+  cover: flutter_libraries_we_love.png
+  link: "https://codemagic.io/flutter-libraries-ebook/"
+  desc: "\"Flutter libraries we love\" focuses on 11 different Flutter library categories. Each category has a list of Flutter libraries as well as a highlighted library that we dig deeper into – including pros and cons, developer's perspective and real-life code examples. The code snippets of all 11 highlighted libraries are updated to their latest versions, which support Flutter 2 with sound null safety."
```

Avoid writing in first person "we dig..." and remember the Oxford comma.  :D I removed "latest versions" because that can go out of date fast.

```suggestion
  desc: "\"Flutter libraries we love\" focuses on 11 different categories of Flutter libraries. Each category lists available libraries as well as a highlighted library that is covered in more detail – including pros and cons, developer's perspective, and real-life code examples. The code snippets of all 11 highlighted libraries support Flutter 2 with sound null safety."
```

---

Thanks so much for the review, changes are made :)